modules/store.py

```diff
@@ -32,4 +35,53 @@ def get_results(db_path='../data/results/manifesto_analysis_results.db', table_n
     """
     conn = sqlite3.connect(db_path)
     results_df = pd.read_sql(f'SELECT * FROM {table_name}', conn)
-    return results_df
\ No newline at end of file
+    return results_df
+
+
+def read_gcs_file(
+        blob_name,
+        bucket_name="llms-as-experts",
+        project="llms-as-experts",
+        encoding="utf-8"
+):
+    """
+    Read file content from Google Cloud Storage
+
+    Args:
+        blob_name (str): The URI of the file, excluding bucket name.
+        bucket_name (str): The name of the bucket.
+        project (str): The name of the project.
+        encoding (str): File encoding.
+
+    """
+    storage_client = storage.Client(project=project)
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(blob_name)
+    with blob.open("r", encoding=encoding) as f:
+        return f.read()
+
+
+def list_gcs_folder(
+        blob_name,
+        bucket_name="llms-as-experts",
+        project="llms-as-experts"
+):
+    storage_client = storage.Client(project=project)
+    return [
+        str(blob.name)
+        for blob in storage_client.list_blobs(bucket_name, prefix=blob_name)
+    ]
```

Comment:
you might want to add the use of the delimiter keyword to future proof agains subfolders as shown here: 
https://cloud.google.com/storage/docs/samples/storage-list-files-with-prefix#storage_list_files_with_prefix-python

also the variable name blob_name is a little misleading, its not the blob name it's the prefix pattern, maybe change to prefix?